btree/btree.c

```diff
@@ -11863,6 +11869,142 @@ static void ut_btree_truncate(void)
 
 	btree_ut_fini();
 }
+
+static void ut_lru_test(void)
+{
+	void                       *rnode;
+	int                         i;
+	int64_t                     mem_after_alloc;
+	int64_t                     mem_init;
+	int64_t                     mem_increased;
+	int64_t                     mem_freed;
+	int64_t                     mem_after_free;
+	struct m0_btree_cb          ut_cb;
+	struct m0_be_tx             tx_data         = {};
+	struct m0_be_tx            *tx              = &tx_data;
+	struct m0_be_tx_credit      cred            = {};
+	struct m0_btree_op          b_op            = {};
+	uint64_t                    rec_count       = MAX_RECS_PER_STREAM*50;
+	struct m0_btree_op          kv_op           = {};
+	struct m0_btree            *tree;
+	struct m0_btree             btree;
+	const struct m0_btree_type  bt              = {
+						     .tt_id = M0_BT_UT_KV_OPS,
+						     .ksize = sizeof(uint64_t),
+						     .vsize = bt.ksize * 2,
+						};
+	uint64_t                    key;
+	uint64_t                    value[bt.vsize / sizeof(uint64_t)];
+	m0_bcount_t                 ksize           = sizeof key;
+	m0_bcount_t                 vsize           = sizeof value;
+	void                       *k_ptr           = &key;
+	void                       *v_ptr           = &value;
+	int                         rc;
+	struct m0_buf               buf;
+	uint32_t                    rnode_sz        = 1024;
```

Comment:
We need to start using 4096 as the rnode size, especially for this ut since our code needs to have the node address to be 4k aligned and also the node size needs to be a multiple of 4k.

Reply:
Done